tests/func_test.py

```diff
@@ -111,6 +144,18 @@ def get_app(self):
             db_conn=None
         )
 
+    def test_post_custom_validator_class(self):
+        """"""
```

Comment:
Don't need this if you're not going to put anything in it. May as well remove it.